indico/modules/events/papers/util.py

```diff
@@ -85,5 +85,9 @@ def has_contributions_with_user_paper_submission_rights(event, user):
     return _query_contributions_with_user_paper_submission_rights(event, user).has_rows()
 
 
-def get_contributions_with_user_paper_submission_rights(event, user):
-    return _query_contributions_with_user_paper_submission_rights(event, user).all()
+def get_user_submittable_contributions(event, user):
+    allowed_states = [PaperRevisionState.to_be_corrected]
+    return (_query_contributions_with_user_paper_submission_rights(event, user)
+            .filter(db.or_(Contribution._paper_last_revision == None,  # noqa
```

Comment:
why `noqa`?

Reply:
`== None` (sqlalchemy does not implement `is_` and `isnot` there)

Reply:
Ah right.